lib/Module/OptNone.cpp

```diff
@@ -0,0 +1,61 @@
+//===-- OptNone.cpp -------------------------------------------------------===//
+//
+//                     The KLEE Symbolic Virtual Machine
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#include "Passes.h"
+
+#include "klee/Config/Version.h"
+
+#include "llvm/IR/Function.h"
+#include "llvm/IR/Instruction.h"
+#include "llvm/IR/Module.h"
+
+#include <set>
+
+namespace klee {
+
+char OptNonePass::ID;
+
+bool OptNonePass::runOnModule(llvm::Module &M) {
+  // Find list of functions that start with `klee_`
+  // and mark all functions that contain a function or invoke call as noneopt
```

Comment:
noneopt -> noneopt

Reply:
`optnone`?

Reply:
100 seconds too late, @251 :)

Reply:
But... your comment  says "noneopt -> noneopt"?!

Reply:
I rely on @MartinNowack reading between the lines :)